### PR TITLE
Add Pactus Glossary

### LINKED ISSUE #135: Terminology
We need to add a terminology part for pactus.org

Examples:
- https://docs.solana.com/terminology
- https://aptos.dev/reference/glossary
- https://docs.ton.org/learn/glossary

### PR DESCRIPTION
**Pull request by kehiy**
This page will help users to find out shore explanations about Pactus-related words like a dictionary and as Pactus glossary to terminology.

In other projects, you can see also crypto-related words, but I decided to add words only related to Pactus here.

In future, we can improve this page.

- Fixes #135

### CODE REVIEW

File: website/learn/glossary/glossary/index.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+# U
+
+## UnBond
+
+An UnBond is a transaction that you unlock staked PAC coins on your validator for withdrawing, whcih
+takes 21 days time.
+
+# V
+
+## Vlidator
```
Comment (Mr-HoDL58) [inline website/learn/glossary/glossary/index.md:67]: validator

Comment (Mr-HoDL58) [inline website/learn/glossary/glossary/index.md:67]: Validator

File: website/learn/glossary/glossary/index.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+A Validator is an entity that participates in the consensus process by sharing its votes.
+
+## Validator Address
+
+Validator address is a unique identifier for validator on Pactus, and they point to a stake balance,
```
Comment (Mr-HoDL58) [inline website/learn/glossary/glossary/index.md:73]: Pactus, they

File: website/learn/glossary/glossary/index.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+# S
+
+## SSPoS (Solid State Proof of Stake)
+
+The Pactus unique consensus algorithm.
+
+## Sortition
+
+The sortition algorithm is responsible for the fair, transparent  and random selection of validators
```
Comment (Mr-HoDL58) [inline website/learn/glossary/glossary/index.md:55]: double space between transparent and

File: website/learn/glossary/glossary/index.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
+---
+layout: learn
+title: Pactus Glossary
+sidebar: Pactus Glossary
+---
+
+<!-- TODO: find a name for pactus merkel root model and add it here (?) -->
+
+# A
```
Comment (b00f) [inline website/learn/glossary/glossary/index.md:9]:
```suggestion
## A
```

and same for others

File: website/learn/glossary/glossary/index.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+## Account
```
Comment (b00f) [inline website/learn/glossary/glossary/index.md:11]:
```suggestion
### Account
```

Comment (b00f) [reply website/learn/glossary/glossary/index.md:11]: For glossary we should have a YML file and the compile the final HTML page based on YML file

**Review by b00f — CHANGES_REQUESTED** [review]

File: website/learn/glossary/glossary/index.md (hunk 4 above)
Comment (kehiy) [reply website/learn/glossary/glossary/index.md:9]: @b00f we will get linting issues with that. we can increase the header level by only 1 for each header!

File: website/learn/glossary/glossary/index.md (hunk 5 above)
Comment (kehiy) [reply website/learn/glossary/glossary/index.md:11]: How we can do that?

Comment (b00f) [reply website/learn/glossary/glossary/index.md:11]: Same as we did in FAQ page. Check that page

**Pull request closed without merging**